### PR TITLE
Donation Modal - made sure rich text string does get rendered as HTML

### PR DESCRIPTION
**Pull request by mmmavis**
Related PRs/issues: #11183

### Test pages
- CMS: https://foundation-s-11183-dona-po14fm.mofostaging.net/cms (creds: `admin2/admin2password`)
- Text with no special rich text element
    - https://foundation-s-11183-dona-po14fm.mofostaging.net/en/privacynotincluded/articles/six-modern-party-from-open-half-do-various/?thank_you=true
- Text with rich text elements (bold text, italics, link)
    - https://foundation-s-11183-dona-po14fm.mofostaging.net/en/campaigns/single-page/?thank_you=true
    - bold text: `healthy internet`
    - italic text: `personal information`
    - link: `UFO`

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-641)

### CODE REVIEW

File: source/js/components/petition/donation-modal.jsx [hunk 1]
```diff
@@ -102,15 +102,22 @@ class DonationModal extends Component {
         </button>
 
         <div className={classNames(`modal-body`, MODAL_BODY_CLASS)}>
-          <h3 className={classNames(`tw-h2-heading`, `text-center`)}>
+          <h3 className={classNames(`tw-h2-heading`, `tw-text-center`)}>
```
Comment (mmmavis) [inline source/js/components/petition/donation-modal.jsx:105]: replacing Bootstrap's `text-center` with Tailwind's `tw-text-center`

File: source/js/components/petition/donation-modal.jsx [hunk 2]
```diff
@@ -102,15 +102,22 @@ class DonationModal extends Component {
...
-          <p className={classNames(`tw-body-large`, `text-center`)}>
-            {this.props.bodyText}
-          </p>
+          <div
+            className={classNames(
+              `tw-rich-text-wrapper`,
+              `tw-body-large`,
+              `tw-text-center`
+            )}
+            dangerouslySetInnerHTML={{
```
Comment (danielfmiranda) [inline source/js/components/petition/donation-modal.jsx:114]: 🤔 Today I learned about `dangerouslySetInnerHTML`!

**Review by danielfmiranda — APPROVED** [review]
Great work @mmmavis! Also, thanks for taking the time to update legacy bootstrap classes with their tailwind equivalent.

**Comment by data-sync-user** [comment]
➤ Simon Acosta Torres commented:

PR has already been merged.

**Pull request merged**